src/Module/Diagnostic/WebFinger.php

```diff
@@ -20,8 +20,6 @@ public static function content()
 			throw $e;
 		}
 
-		$app = self::getApp();
-
 		$addr = defaults($_GET, 'addr', '');
 		$res = '';
```

Comment:
Code standards: Please normalize quoting style in this file (this line is fine but I can't attach this comment to the relevant line).